editor/code_editor.cpp

```diff
@@ -1095,10 +1095,14 @@ void CodeTextEditor::clone_lines_down() {
 	text_editor->update();
 }
 
-void CodeTextEditor::goto_line(int p_line) {
+void CodeTextEditor::goto_line(int p_line, bool p_center_to_cursor) {
```

Comment:
I see that ScriptTextEditor has `void ScriptTextEditor::goto_line(int p_line, bool p_with_error)`, so it seems a bit confusing to have this method with a different optional boolean parameter in those two related classes.

Might be better to add `CodeTextEditor::goto_line_centered` that calls `goto_line` and then `text_editor->call_deferred("center_viewport_to_cursor");`.